documentation/images/plantuml/boardfarm_prplMesh_classes.puml

```diff
@@ -86,7 +86,7 @@ package prplMesh_entities <<Frame>> {
         check_log()
```

Comment:
This is called wait_for_log instead of check_log. Also, it's an abstract function.

For radios, there is no accessor function. The relation with the Radio class is sufficient.

Also the class is ALEntity, not prplmesh_agent.